src/ontology/cl-edit.owl

```diff
@@ -25654,13 +25655,14 @@ SubClassOf(obo:CL_1000346 obo:CL_0002071)
 
 # Class: obo:CL_1000347 (colonocyte)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "PMID:34497389") obo:IAO_0000115 obo:CL_1000347 "A columnar cell of the colon that is part of the colonic epithelium.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "PMID:34497389") Annotation(oboInOwl:hasDbXref "PMID:16870803") Annotation(oboInOwl:hasDbXref "PMID:23821742") obo:IAO_0000115 obo:CL_1000347 "A columnar cell that is part of the colonic epithelium. This cell is responsible for the absorption, transport, and metabolization of short-chain fatty acids (SCFAs) produced by gut bacteria, as well as the transport and absorption of water and electrolytes.")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_1000347 "FMA:263114")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_1000347 "enterocyte of colon")
 AnnotationAssertion(oboInOwl:hasNarrowSynonym obo:CL_1000347 "vacuolar absorptive cell of epithelium of colon")
 AnnotationAssertion(rdfs:label obo:CL_1000347 "colonocyte")
 EquivalentClasses(obo:CL_1000347 ObjectIntersectionOf(obo:CL_0002071 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000397)))
-SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_1000347 obo:CL_0002071)
+SubClassOf(obo:CL_1000347 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0006833))
```

Comment:
This looks like a better term to me https://www.ebi.ac.uk/QuickGO/term/GO:0035377